### ISSUE TITLE
"No module named pymongo.connection" error

### ISSUE THREAD

**Issue by yusufyilmaz** [airnotifier/airnotifier#139]
Hello There,

I am having some problem installing Airnotifer for Moodle on Aws EC2 free tier installiation.

I have successfully installed airnotifier when python version was 2.7.6. Airenotifier server was built up and I was able to access http://my.ip.add.ress:8801 this worked fine (without ssl). I was accessing the web control panel of airnotifer. However, my device was not registering to airnotfier but registerig moodle was fine. I saw device token in moodle db. And I wanted to give Airnootifier a try wheter it sends the notfication to my android app (I have installed GCM project number and api key in airnotifier).  However I was unable to send push notification to Android. 

And I thought this is because there may be the reason that there is no ssl server on my airnotifer server. and I went to letsencrypt and installed ssl to my subdomian. And in conf file of airnotifer I made the required settings

> https = True
> httpscert = "/etc/letsencrypt/live/aws.mydomian.com/fullchain.pem"
> httpskey = "/etc/letsencrypt/live/aws.mydomian.com/key.pem"
> 
> port = 443

After this I run 

`sudo python airnotifier.py >> airnotifier.log 2>> airnotifier.err`

But nothin happened and checked the err file and it says ssl 

> File "airnotifier.py", line 285, in <module>
>     AirNotifierApp(services=services).main()
>   File "airnotifier.py", line 193, in main
>     ssl_ctx = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
> AttributeError: 'module' object has no attribute 'create_default_context'

I have made some search and I have seen some posts that is because of the version of python. which was adviced to upgrade t python 2.7.9 and I did.

After this ssl error is lost but now I am getting this annoying error

> Traceback (most recent call last):
>   File "airnotifier.py", line 31, in <module>
>     from pymongo.connection import Connection
> ImportError: No module named pymongo.connection

I have made very broad research about it nut no lack to find any solution. So where is the problem. Can you figure out? or is this just not suitable for python version or so?

**Comment by dcai** [comment]
It appears you haven't reinstall pymongo after upgrade to 2.7.6. Use:

    sudo pip install -r requirements.txt

**Comment by yusufyilmaz** [comment]
I have already tried that reinstall everythin. I have tried with other options as well:
`sudo pip install -U -r requirements.txt`
`sudo pip install -I -r requirements.txt`

I have tried above codes just to be sure everything reinstalled and upgraded but no luck. 

I am running 
`python install.py`

but there is no chacne. Still I get this

![image](https://cloud.githubusercontent.com/assets/4222204/13557624/339422b6-e3fd-11e5-923c-0e59acf51cc6.png)

**Comment by yusufyilmaz** [comment]
Quick update for the issue. This error stills continue on ubuntu. Because Ubuntu comes with Python default 2.7.6 version on AWS. When I upgrade the version to 2.7.9, this error continues. However, I launch an instace with Debian Jessie which comes with Python2.7.9  default. So I did not get error. IT worked like a charm. Maybe I am not updating the Python on Ubuntu in the right way. I used [this ](https://www.ralphlepore.net/installing-python-279-on-ubuntu-1404-lts/) procedure to update the Python on Ubuntu.
